backend/src/main/java/com/bakdata/conquery/io/xodus/NamespacedStorageImpl.java

```diff
@@ -122,10 +127,15 @@ protected void createStores(Collector<KeyIncludingStore<?, ?>> collector) {
 
 
 		collector
-			.collect(dataset)
-			.collect(dictionaries)
-			.collect(concepts)
-			.collect(imports);
+			.collect(environment,dataset)
```

Comment:
irgendwie wirkt das etwas redundant?

Reply:
Ich splitte das in einen Store und einen Environment collector okay?

Reply:
Okay, ich habs jetzt doch anders gemacht damit man mehr möglichkeiten zum hinzufügen hat